src/and-cli-github-issue.ts

```diff
@@ -0,0 +1,252 @@
+            // Replace escaped new lines/tabs w/ actual chars
+            dto.body = dto.body.replace(/\\n/g, "\n").replace(/\\t/g, "\t");
+        }
+
+        if (StringUtils.isEmpty(dto.body)) {
+            const terminatorPhrase = "/end";
+            dto.body = (
+                await Prompt.multiline(
+                    `Enter a body for this issue: ('${terminatorPhrase}' to finish input)`,
+                    terminatorPhrase
+                )
+            ).join("\n");
+        }
+
+        return dto as CreateIssueDto;
+    };
+
+    const validatePatternOrExit = (value: string): boolean | never => {
+        if (value.match(`${Github.andcultureOrg}.*#[0-9]+`)) {
+            return true;
+        }
+
+        Echo.error(
+            `Value provided for ${CLONE_FROM_OPTION.option} does not match an ${Github.andcultureOrg} ${REPO_ISSUE_EXAMPLE} pattern.`
+        );
+
+        shell.exit(1);
+    };
+
+    // #endregion Functions
+
+    // -----------------------------------------------------------------------------------------
+    // #region Entrypoint
+    // -----------------------------------------------------------------------------------------
+
+    program
+        .description(CommandDefinitions.github.children.issue.description)
+        .option(
+            ADD_OPTION.toString(),
+            `Add issue to a repository (optionally used in conjunction with ${REPO_OPTION.option}, ${TITLE_OPTION.option}, and ${BODY_OPTION.option})`
+        )
+        .option(
+            CLONE_FROM_OPTION.toString(),
+            `Clone an existing issue to the target repo (used in conjunction with ${REPO_OPTION.option})`
+        )
+        .option(
+            BODY_OPTION.toString(),
+            `Set the body for an issue to be created with ${ADD_OPTION.option} (will be prompted if not provided via cli)`
+        )
+        .option(
+            LIST_OPTION.toString(),
+            `List issues for a given repo (used in conjunction with ${REPO_OPTION.option})`
+        )
+        .option(
+            REPO_OPTION.toString(),
+            `Repository name to act on (used in conjunction with ${ADD_OPTION.option} or ${LIST_OPTION.option}, for example)`
+        )
+        .option(
+            TITLE_OPTION.toString(),
+            `Set the title for an issue to be created with ${ADD_OPTION.option} (will be prompted if not provided via cli)`
+        )
+        .parse(process.argv);
+
+    const {
+        add: addIssue,
+        body,
+        cloneFrom,
+        list,
+        title,
+        repo,
+    } = program.opts();
+
+    const cloneIssueFromRepo = StringUtils.hasValue(cloneFrom);
+    const listIssuesByRepo = list === true && StringUtils.hasValue(repo);
+
+    if (listIssuesByRepo) {
+        const issues = await Github.getIssues(Github.andcultureOrg, repo);
+        print(repo, issues);
+        return;
+    }
+
+    if (addIssue === true) {
+        const dto = await promptForNewFields({ repo, body, title });
```

Comment:
I'm might be incorrect but I don't see where you are declaring this method as async. It might be on line 125 but just double checking.

Reply:
Yep - that method is declared async on line 125 due to the async prompt, and the body of the whole function we're in right now is declared async on line 19.